generated/provider_dependencies.json

```diff
@@ -28,6 +28,10 @@
       "boto3>=1.28.0",
       "botocore>=1.31.0",
       "jsonpath_ng>=1.5.3",
+      "mypy-boto3-appflow>=1.24.0",
+      "mypy-boto3-rds>=1.24.0",
+      "mypy-boto3-redshift-data>=1.24.0",
+      "mypy-boto3-s3>=1.24.0",
```

Comment:
I wonder where did this come from?
this file is auto generated by pre-commits.
maybe you used another version of provider.yaml when you run the tests?

Reply:
Good question. Let me check through things the only dependency defined that I removed was OpenSearch-dsl because after 2.1.0 of OpenSearch-py that was added in directly to have a single package.